queries.sql

```diff
@@ -81,4 +81,59 @@ WHERE owners.full_name = 'Dean Winchester' AND animals.escape_attempts = 0;
 
 SELECT owners.full_name, COUNT(animals.name) FROM animals 
 JOIN owners ON animals.owner_id = owners.id 
-GROUP BY (owners.full_name);
\ No newline at end of file
+GROUP BY (owners.full_name);
+
+SELECT visits.date_of_visit, animals.name, vets.name FROM visits
+LEFT JOIN animals ON visits.animal_id = animals.id
+RIGHT JOIN vets ON visits.vet_id = vets.id
+WHERE vets.name = 'William Tatcher'
+ORDER BY visits.date_of_visit;
+
+SELECT animals.id, vets.name, animals.name FROM visits
+JOIN animals ON visits.animal_id = animals.id
+LEFT JOIN vets ON visits.vet_id = vets.id
+WHERE vets.name = 'Stephanie Mendez';
+
+SELECT vets.name AS vet_name, species.name as species FROM specialization
+JOIN species ON specialization.species_id = species.id
+RIGHT JOIN vets ON specialization.vet_id = vets.id;
+
+SELECT animals.id, vets.name, animals.name, date_of_visit FROM visits
+JOIN vets ON visits.vet_id = vets.id
+JOIN animals ON visits.animal_id = animals.id
+WHERE vets.name = 'Stephanie Mendez' 
+AND date_of_visit BETWEEN '2020-04-01' AND '2020-08-30';
+
+SELECT animal_id, animals.name, COUNT(animal_id) FROM visits
+JOIN animals ON visits.animal_id = animals.id
+GROUP BY animal_id, animals.name
+order by COUNT(animal_id);
+
+SELECT animal_id, animals.name, vets.name AS vet_name, date_of_visit FROM visits
```

Comment:
- [ ] Here, you are supposed to single out that one animal that visited Maisy Smith first. You instead listed out each animal that visited Maisy Smith in ascending order. Here you can use the command `ASC
  LIMIT 1;
` at the end of the query to fix that :+1: